MODULE.bazel

```diff
@@ -43,6 +44,7 @@ use_repo(bazel_lib_toolchains, "yq_linux_arm64")
 use_repo(bazel_lib_toolchains, "yq_linux_ppc64le")
 use_repo(bazel_lib_toolchains, "yq_linux_s390x")
 use_repo(bazel_lib_toolchains, "yq_windows_amd64")
+use_repo(bazel_lib_toolchains, "yq_windows_arm64")
```

Comment:
This requires https://github.com/bazel-contrib/bazel-lib/pull/1125 to be merged first.

Reply:
Currently waiting for a new release so that we can bump the version here.